irteus/BULLET/Makefile.Linux

```diff
@@ -0,0 +1,19 @@
+CC = c++ -O2
+OBJOPT = -o 
+OUTOPT = -o 
+LD = c++
+SOFLAGS = -shared
+EXELD = c++
+EXESFX = 
+LSFX = so
+LPFX = lib
+LIBS = -L$(ARCHDIR) -lRAPID 
+
+
+ifeq ($(shell /bin/uname -m), x86_64)
+CC += -m32 
+LD += -m32
+EXELD += -m32 
+endif
```

Comment:
currently we do not check `i386` on travis, but do you need this? do we still running euslisp on 32bit intel machine?

Reply:
As far as I know, we do not need intel 32bit. This `Makefile.Linux` is just copied from [PQP/Makefile.Linux](https://github.com/euslisp/jskeus/blob/master/irteus/PQP/Makefile.Linux).
If these lines should be removed, I can. But it may be beyond this PR, and we can discuss this in new Issue.